Package.swift

```diff
@@ -231,7 +231,7 @@ func addResolvedTargets() {
 
 addDependencies(
     clientRuntimeVersion: "0.37.0",
-    crtVersion: "0.20.0"
+    crtVersion: "0.22.0"
```

Comment:
Q: was this change made by codegen after changing the version in `packageDependencies.plist`?

Reply:
replied offline, Package.swift should be generated based on packageDependencies.plist so changing this is fine